# URA0012: U# does not support throwing exceptions

UdonSharp does not support throwing exceptions since Udon does not have support for exception throwing at the moment

## Examples

```csharp
using System;

using UdonSharp;

using UnityEngine;

class ExampleClass : UdonSharpBehaviour
{
    [SerializeField]
    private GameObject go;

    private void Start()
    {
        if (go == null)
            throw new InvalidOperationException(); // URA0012
    }
}
```

## Solution

```csharp
using System;

using UdonSharp;

using UnityEngine;

class ExampleClass : UdonSharpBehaviour
{
    [SerializeField]
    private GameObject go;

    private void Start()
    {
        if (go == null)
            return;
    }
}
```
