# URA0053: Invalid target property for FieldChangeCallbackAttribute

Invalid target property for FieldChangeCallbackAttribute

## Examples

```csharp
using UdonSharp;

class ExampleClass : UdonSharpBehaviour
{
    [FieldChangeCallback("Property")]
    private string _bkProperty; // URA0053
}
```

## Solution

```csharp
using UdonSharp;

class ExampleClass : UdonSharpBehaviour
{
    // create a new property or specify existing property
    [FieldChangeCallback("Property")]
    private string _bkProperty;

    public string Property
    {
        set => _bkProperty = value;
        get => _bkProperty;
    }
}
```

## CodeFixes

![](https://user-images.githubusercontent.com/10832834/125989531-8850919d-395f-4127-84c1-b4e3f5387784.gif)
